AutomationHelpers/src/UserCodeCollections/SystemLibrary.cs

```diff
@@ -85,5 +85,16 @@ public static TimeSpan StopTimer(string timerName)
 
             throw new Exception(string.Format("Timer '{0}' does not exist.", timerName));
         }
+
+        /// <summary>
+        /// Return current time as a string in the given format
+        /// </summary>
+        /// <param name="expectedFormat">The string format which is used for the return value. Default is dd.MM.yyyy</param>

```

Comment:
The format the time and date are returned in. Default = dd.MM.yyyy
What about seconds, minutes, hours?